test/utilities/apexUtils.ts

```diff
@@ -178,3 +177,78 @@ export async function createAnonymousApexFile(): Promise<void> {
   await textEditor.save();
   await pause(1);
 }
+
+export async function createVisualforcePage(): Promise<void> {
+  const workbench = await browser.getWorkbench();
+  let textEditor: TextEditor;
```

Comment:
`textEditor` isn't needed until line 202.  Could you remove line 183, and then add 'let' to line 202?